x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -339,7 +296,7 @@ describe('AllCasesListGeneric', () => {
     useGetCasesMock.mockReturnValue({
       ...defaultGetCases,
       filterOptions: { ...defaultGetCases.filterOptions, status: CaseStatuses.closed },
-      selectedCases: [...useGetCasesMockState.data.cases, collectionCase],
```

Comment:
just a note to remove mock data regarding sub-cases. I haven't gone through the whole PR so apologies if you are doing it.

Reply:
I think I got them all, let me know if I missed some 😬